low-level/solution/lr2/storage/health.py

```diff
@@ -110,6 +111,7 @@ def get_data(self, rpath):
             logger.error(self.log.svc_log(f"{msg}"))
             raise ResourceMapError(errno.EINVAL, msg)
 
+        info = Utility.normalize_kv(info, "NA", "Not Available")
```

Comment:
Better this can take list of undesired values to normalize.
```
info = Utility.normalize_kv(
     info,
     const.RESOURCE_MAP["undesired_values"],
     "Not Available")
```

Reply:
Yes, Thanks mariyappan. it will increase the functionality of that function is used.
I will address this in the next commit.

Reply:
Done